src/hooks/ui/useCustomTheme.ts

```diff
@@ -183,6 +185,41 @@ export const useCustomTheme = () => {
              * Below here - custom components implemented by ShapeDiver 
              */
 
+			/**
+			 * AppBuilderContainerWrapper
+			 * 
+			 * Used for defining theme overrides per AppBuilder container.
+			 */
+			AppBuilderContainerWrapper: AppBuilderPageThemeProps({
+				containerThemeOverrides: {
+					/** Theme overrides for the "top" container. */
+					top:  {
+						components: {
+							Paper: Paper.extend({
+								defaultProps: {
+									withBorder: false,
+								}
+							}),
+							AppBuilderHorizontalContainer: {
+								defaultProps: {
+									justify: "left"
+								}
+							},
+							AppBuilderTextWidgetComponent: AppBuilderTextWidgetThemeProps({
+								styles: { root: { overflow: "clip" }}
+							}),
+						}
+					},
+					/** Theme overrides for the "bottom" container. */
+					bottom: {
+						components: {
+							AppBuilderTextWidgetComponent: AppBuilderTextWidgetThemeProps({
+								styles: { root: { overflow: "auto" }}
+							}),
+						}
+					}
```

Comment:
@mathieu-shapediver in the bottom container, we use scrollbars for the text widgets